linux_os/guide/system/permissions/mounting/kernel_module_usb-storage_disabled/rule.yml

```diff
@@ -40,6 +40,7 @@ references:
     cobit5: APO13.01,DSS01.04,DSS05.03,DSS05.04,DSS05.05,DSS05.07,DSS05.10,DSS06.03,DSS06.10
     iso27001-2013: A.11.2.6,A.13.1.1,A.13.2.1,A.18.1.4,A.6.2.1,A.6.2.2,A.7.1.1,A.9.2.1,A.9.2.2,A.9.2.3,A.9.2.4,A.9.2.6,A.9.3.1,A.9.4.2,A.9.4.3
     cis-csc: 1,12,15,16,5
+    cis@rhel8: "1.1.23"
```

Comment:
Remove quotes here... greater than 1 decimal point don't need quotes as it is no longer treated as a decimal in yaml.

Reply:
fixed (note earlier feedback said to add the quotes)

Reply:
fixed